### ISSUE TITLE
total loss divided by N

### ISSUE THREAD

**Issue by bysowhat** [balancap/SSD-Tensorflow#35]
hi:
  the loss function is dived by N in ssd paper, where N is the number of matched default boxes.But i can not find this division operation in your code.Can you help me  to figure it out please?

**Comment by balancap** [comment]
I use the function `tf.losses.compute_weighted_loss` with a boolean mask corresponding matched boxes. The former automatically renormalises the loss with the number of non-zero entries in the mask.

**Comment by bysowhat** [comment]
thank you very much!you did help me a lot :-)
